examples/bri-3/.eslintrc.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    project: 'tsconfig.json',
+    tsconfigRootDir : __dirname, 
+    sourceType: 'module',
+  },
+  plugins: ['@typescript-eslint/eslint-plugin'],
+  extends: [
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettier/recommended',
+  ],
+  root: true,
+  env: {
+    node: true,
+    jest: true,
+  },
+  ignorePatterns: ['.eslintrc.js'],
+  rules: {
+    '@typescript-eslint/interface-name-prefix': 'off',
```

Comment:
is turning these off by default or some custom rules?

Reply:
default. we can agree on the rules as part of #492

Reply:
> great start... we need to document the usage of interfaces vs classes. I think this should be driven by whether the class is stateless or stateful ... for stateful you probably want to instantiate the object and load the data into it, vs invoking a service that just processes data that can be invoked at run time.
> 
> Or am I missing something?
> 
> @ognjenkurtic @skosito @Kasshern @Manik-Jain

Thx. See my answer above re: interfaces vs classses. Classes seems like the best option for domain objects. We ll continue the discussion in parallel as we move forward.